resume.html

```diff
@@ -0,0 +1,515 @@
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+                    <h5>System Developer</h5>
+                    <p>AL_Rowad for IT Solutions</p>
+                    <p>UAE - Abu Dhabi</p>
+                </div>
+                <div class="col-4">
+                    <p> Designed and developed a web-based application “e-Correspondence System” using mod-plsql Oracle and web technologies
+                    (html, css, JavaScript, and Ajax), used by more than 10,000 users in government organizations, such as the Abu Dhabi
+                    Police and the Ministry of Interior among others.</p>
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row narrow-row">
+                <div class="col gray-div">
+            
+                </div>
+            </div>
+            <div class="row normal-row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-8 sub-title">
+                    Jan 2007 - Jun 2008
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+                    <h5>System Developer</h5>
+                    <p>Kantakji Center - KedySoft</p>
+                    <p>Aleppo, Syria</p>
+                </div>
+                <div class="col-4">
+                    <p> Created and developed solutions and interactive websites using C#.NET 2005, ASP.NET 2005 & MSSQL Server 2005.</p>
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row narrow-row">
+                <div class="col gray-div">
+            
+                </div>
+            </div>
+            <div class="row normal-row">
+                <div class="col-2 gray-div">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-4">
+            
+                </div>
+                <div class="col-2 gray-div">
+            
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-2 gray-div">
+            
+                </div>
```

Comment:
All these divs just to create space 😱 You don't need to do this.
You should only use HTML elements to wrap content (apart from a few exceptions). Bootstrap offers everything you need to create blank space ([offset](https://getbootstrap.com/docs/4.0/layout/grid/#offsetting-columns)), and also you don't need to have a separate `row` for everything. You can have several `col-12` blocks inside one `row` and they will just sit on top of each other 🙂